src/integrationtest/docker/tomcat/conf.yml

```diff
@@ -0,0 +1,20 @@
+packaging:
+  add:
+    - target/addon-self-administration-${project.version}
```

Comment:
I can not because the assets need to be in the classpath of the self-administration